tests/testthat/test-checks.R

```diff
@@ -15,4 +15,165 @@ test_that("Checks work", {
     .check_generation_time_valid(function(x) 3),
     "Must have length"
   )
+  expect_error(
```

Comment:
maybe add some comments here for code reviewers to see what is being checked? It's a bit hard to see with so many arguments. Alternatively define a default function to avoid having to repeat the same arguments over and over again. See here for an example of this approach:
https://github.com/epiforecasts/EpiNow2/blob/df1fdc8bcb361da308037fbb9c8deb6f3fc45235/tests/testthat/test-estimate_infections.R#L7

Reply:
Thanks for pointing me to this. I've restructured the tests here 31d09158e831a52233932b3652ad2fbc2c0de33b.

Reply:
Awesome, this is much clearer.